### DISCUSSION TITLE
[Dropdown] Margin between dropdown content menus

### DISCUSSION THREAD

**Discussion by skarensmoll** [radix-ui/primitives#1611] [Help] [answered]
Hi there !

I was wondering if there's a prop to change the margin between dropdown content and submenus:

![Screenshot 2022-08-11 at 11 44 48](https://user-images.githubusercontent.com/7997951/184107312-500c2c12-e32d-4ec9-b6d7-1f50a2052b28.png)

Currently we have for example sideOffset, to customize distance in px from the trigger but seems for distance between menus/submenus is not possible (?)

**Comment by andy-hook** [comment, marked as answer]
Hey @skarensmoll the `SubContent` part also accepts the `sideOffset` prop which behaves the same way.

https://www.radix-ui.com/docs/primitives/components/dropdown-menu#subcontent

https://codesandbox.io/s/friendly-goldstine-b0h2pg?file=/App.js

Is that what you meant?

**Reply by skarensmoll** [reply]
Hey @andy-hook thanks for your fast reply! yes that is what I meant! actually I am using version `0.1.6` so I dont think i can use subcontent's sideOffset yet :(

**Reply by skarensmoll** [reply]
I used [data-side] props in the meantime to change the spacing, but once I upgrade the version will use the sideOffset prop 😃

**Reply by andy-hook** [reply]
Oh I see! sideOffset should still work for `0.1.6`, the main difference is that you'll be using the nesting api instead of the new `SubContent` part, here's the example on `0.1.6`:

https://codesandbox.io/s/delicate-fast-r5owuk?file=/App.js

**Reply by skarensmoll** [reply]
ohhh I see thanks a lot!!! @andy-hook
